style.css

```diff
@@ -0,0 +1,187 @@
+.main {
+  width: 375px;
+  height: 6968px;
+  background: #fff;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  order: 0;
+  left: 0;
+  top: 0;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  order: 0;
+  left: 0;
+  top: 0;
+  width: 375px;
+  height: 48px;
+  background: #3c3a39;
+}
+
+.placeholder {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  width: 101px;
+  height: 25px;
+  left: 24px;
+  border-radius: 8px;
+  margin-top: 0;
+}
+
+#text {
+  width: 100px;
+  height: 24px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #fff;
+  display: flex;
+  flex: none;
+  order: 0;
+  margin-top: 20px;
+  align-items: center;
+  text-align: center;
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  padding: 2px;
+  top: -24px;
+  position: absolute;
+  width: 25px;
+  height: 25px;
+  left: 320px;
+  border-radius: 8px;
+  margin-top: 30px;
+}
+
+#menu-icon {
+  position: absolute;
+  display: flex;
+  width: 18px;
+  height: 12px;
+  right: 6px;
+  margin-top: -2px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 590px;
+  background: url('../Images/a.png') no-repeat #1c1a19;
+  background-position: 100% 100%;
+  order: 1;
+}
+
+.head {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+}
+
+.primary-text {
+  width: 327px;
+  height: 124px;
+  display: flex;
+  margin-left: 24px;
+  margin-top: 120px;
+  font-family: 'Crete-Round', sans-serif;
```

Comment:
Good job for using the proper font name, but you forget to include the font in your `head` tag. That's why this font is not applied. You can take it from the [google fonts](https://fonts.google.com/specimen/Crete+Round?query=Crete+Round).

![image](https://user-images.githubusercontent.com/34779928/198818205-bccf0752-aeac-499a-9413-d5e542a27c4f.png)